Dockerfile.search

```diff
@@ -1,6 +1,16 @@
-FROM getmeili/meilisearch:latest
+FROM getmeili/meilisearch:v1.14
 
-ARG SEARCH_DB
+ARG GOOGLE_CLOUD_CLI_VERSION=522.0.0
+
+RUN <<EOF
+apk add --no-cache python3
+curl --silent https://storage.googleapis.com/cloud-sdk-release/google-cloud-cli-${GOOGLE_CLOUD_CLI_VERSION}-linux-x86_64.tar.gz | tar zxf - -C /
```

Comment:
untarrring into / always makes me a little OCD.

Can I suggest

```
mkdir /google-cloud-sdk
....| tar --strip-components=1 -C /google-cloud-sdk
```

that should be exactly the same as currently but robust to someone in the future adding random stuff into the root of the tar.

If could even untar into ~ or /workspace

Reply:
Sure. `/usr/local/google-cloud-sdk` or `/opt/google-cloud-sdk`? It doesn't have to be `/`.